<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="testing.GameplayTest" tests="7" skipped="0" failures="1" errors="0" timestamp="2023-03-24T16:33:34" hostname="AB-ASUS-1" time="0.174">
  <properties/>
  <testcase name="TestServingStationServeCustomerLettuceBurger" classname="testing.GameplayTest" time="0.071">
    <failure message="java.lang.AssertionError: Error: correct recipes are not being accepted due to Recipe.matchesRecipe expected [true] but found [false]" type="java.lang.AssertionError">java.lang.AssertionError: Error: correct recipes are not being accepted due to Recipe.matchesRecipe expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:97)
	at org.testng.Assert.failNotEquals(Assert.java:969)
	at org.testng.Assert.assertTrue(Assert.java:43)
	at testing.GameplayTest.TestServingStationServeCustomerLettuceBurger(GameplayTest.java:129)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at testing.GdxTestRunner.render(GdxTestRunner.java:57)
	at com.badlogic.gdx.backends.headless.HeadlessApplication.mainLoop(HeadlessApplication.java:128)
	at com.badlogic.gdx.backends.headless.HeadlessApplication$1.run(HeadlessApplication.java:93)
</failure>
  </testcase>
  <testcase name="TestServingStationServeCustomerLettuceOnionSalad" classname="testing.GameplayTest" time="0.003"/>
  <testcase name="TestServingStationServeCustomerPlainBurger" classname="testing.GameplayTest" time="0.0"/>
  <testcase name="TestBuyShopItem" classname="testing.GameplayTest" time="0.001"/>
  <testcase name="TestServingStationServeCustomerLettuceTomatoSalad" classname="testing.GameplayTest" time="0.0"/>
  <testcase name="TestGoldSetandGetBalance" classname="testing.GameplayTest" time="0.001"/>
  <testcase name="TestServingStationServeCustomerOnionTomatoSalad" classname="testing.GameplayTest" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
